src/components/FeatureViews/Accounts/ProfileHeading.js

```diff
@@ -190,9 +203,16 @@ class ProfileHeading extends Component {
                 </>
             );
         }
+
     }
 
+handleClick(){
+    this.setState({open:true,})
```

Comment:
Add a new line before this line with this line:
```
event.preventDefault();
```

Reply:
binding the button to onclick={this.handleClick().bind(this)}sets all buttons to the same function, is there a way for me to bind it to only one button?

Reply:
Rename your function? I'm not quite certain what you're asking specifically. 

I also just realized this, but you're not using a react hook. React Hooks will make your life significantly easier. Here are some potential references:
- [Egghead tutorial on refactoring react component class to hook](https://egghead.io/lessons/react-refactor-a-class-component-with-react-hooks-to-a-function)
- [Blog post on the same topic](https://levelup.gitconnected.com/refactoring-react-class-components-with-hooks-d15abb7fe71a)

Reply:
I was trying to use hooks, I found the problem was the function was not capitalized, and therefore not rendering as a hook.

Reply:
I'm having some trouble getting the dialog to close, I've pushed up the latest updates I've made so you can take a look.

Reply:
Error is this: 
![image](https://user-images.githubusercontent.com/52120602/73628338-daa0b480-4604-11ea-93d0-7419b93b1cd7.png)

Reply:
You need to declare state variables as:

    const [dialog, setDialog] = useState(props.open);
    const [date, setDate] = useState(null);

[useState](https://reactjs.org/docs/react-component.html#setstate) returns `[value, setter]`, where `value` is the current value of the state variable, and `setter` is how you set the state variable.